d2l-scroll-wrapper.html

```diff
@@ -169,14 +170,13 @@
 			:host([is-rtl]) .left,
 			.right {
 				position: absolute;
-				right: 0;
+				right: 32px;
```

Comment:
This moves the arrow buttons inside the table itself, for all tables not just ones with a sticky header. Is design OK with this change?

Reply:
This would also change every table in the LMS. Maybe replace this with a mixin or variable

Reply:
Undoing this change as it's been decided that the chevrons will be hidden for sticky tables.